src/Core/Security/Domain/Provider/OpenIdProvider.php

```diff
@@ -679,7 +723,8 @@ private function logExceptionInLoginLogFile(string $message, \Exception $e): voi
         $this->centreonLog->insertLog(
             CentreonUserLog::TYPE_LOGIN,
             sprintf(
-                "[Openid] [Error] $message",
+                "[Openid] [Error] %s, [%s]: %s",
+                $message,
```

Comment:
some `$message` value have already `%s`, so the number of sprintf bound values will be wrong